homeassistant/components/demo/vacuum.py

```diff
@@ -48,11 +48,11 @@
 )
 
 FAN_SPEEDS = ["min", "medium", "high", "max"]
-DEMO_VACUUM_COMPLETE = "0_Ground_floor"
-DEMO_VACUUM_MOST = "1_First_floor"
-DEMO_VACUUM_BASIC = "2_Second_floor"
-DEMO_VACUUM_MINIMAL = "3_Third_floor"
-DEMO_VACUUM_NONE = "4_Fourth_floor"
+DEMO_VACUUM_COMPLETE = "Demo Vacuum 0_Ground_floor"
```

Comment:
Please either use underscore and no space or just space between words. Don't mix.